smoketest/webhook_test.go

```diff
@@ -0,0 +1,73 @@
+package smoketest
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/target/goalert/smoketest/harness"
+)
+
+type WebhookTestingAlert struct {
+	AlertID   int
+	AlertType string
+	Summary   string
+	Details   string
+}
+
+func TestWebhookAlert(t *testing.T) {
+	t.Parallel()
+
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		var alert WebhookTestingAlert
```

Comment:
Send this into a channel so we can wait for it below, and then assert on the fields in the test instead of this handler